src/sessions/softphone-session-handler.ts

```diff
@@ -360,12 +363,28 @@ export class SoftphoneSessionHandler extends BaseSessionHandler {
 
     currentEmittedEvent.activeConversationId = this.determineActiveConversationId(session);
 
-    this.log('debug', 'emitting `conversationUpdate`', { event, previousEmittedEvent: this.lastEmittedSdkConversationEvent, currentEmittedEvent, sessionId: session?.id }, { skipServer: true });
+    const previousEventForLogging = this.pruneConversationUpdateForLogging(this.lastEmittedSdkConversationEvent);
+    const currentEventForLogging = this.pruneConversationUpdateForLogging(currentEmittedEvent);
+    this.log('debug', 'emitting `conversationUpdate`', { event, previousEmittedEvent: previousEventForLogging, currentEmittedEvent: currentEventForLogging, sessionId: session?.id }, { skipServer: true });
     this.lastEmittedSdkConversationEvent = currentEmittedEvent;
 
     this.sdk.emit('conversationUpdate', currentEmittedEvent);
   }
 
+  private pruneConversationUpdateForLogging (update: ISdkConversationUpdateEvent): unknown {
```

Comment:
NIT: Can we add a return proper type instead of `unknown`?

Reply:
Yes, we can. I was thinking I couldn't because I was removing `session` in the `IStoredConversationState`s, but forgot that it's ok because that's optional in that type.